slides/week2/git.org

```diff
@@ -0,0 +1,88 @@
+#+TITLE: GitHub Training Two 2017
+#+AUTHOR: Sahit Chintalapudi, Josh Ting 
+#+EMAIL: schintalapudi@gatech.edu, josh.ting@gatech.edu
+
+* Let's Recap
+- We have a fork of the software training repo on our local GitHub account
+- Reminder: a fork is an independently maintained repository based off of another
+- The original repository is referred to as "upstream"
+
+* Things have changed upstream, what do we do?
+- We need a series of git commands that can update our fork and local repo with the new content in the upstream repo
+- All terminal git commands are in the form:
+  #+BEGIN_SRC shell
+    git <command> <param1> <param2> ...
+  #+END_SRC
+
+* Git pull
+- First, navigate to the git repo from the terminal
+- We can keep our fork up to date with the /git pull/ command
+
+  #+BEGIN_SRC shell
+    cd software-training    # or wherever your repo is
+    git pull upstream master
+  #+END_SRC
+
+- What does this mean?
+      - /git pull/: pull from one GitHub repository into our current repository
+      - /upstream/: specify that we are pulling from the upstream repository
+      - /master/: pull into the "master" branch of our current repository
+- What are branches? That's classified! Stay tuned for next lesson
+# Basically say that this is outside the scope of the current lesson and if they want to read up on it on their own we won't stop them
```

Comment:
I think that if the remote branch is the same as the local, you can specify `git pull upstream`. That depends on your gitconfig though (`push.default`), so I would lean towards keeping it this way. Just letting you know that could be an option if we want to ignore branches completly right now.